brain/compartmentReportView.h

```diff
@@ -78,6 +78,22 @@ class CompartmentReportView
      */
     BRAIN_API std::future<brion::Frames> load(double start, double end);
 
+    /** Load frames between start and end time stamps.
+     *
+     * @param start the start time stamp
+     * @param end the end time stamp
+     * @param stride the time stride
```

Comment:
I don't care very much a bout num or count, but I'd like it to be the same in both C++ and Python (except for camelCase versus underscores). For other parts of the wrapping it's not a big issue, but this one in particular changes quite a lot. I wouldn't mind the names different if the Python names are documented in CompartmentReport::getMetaData (and only for the Python docs), actually I think this should be done anyway. @kaabimg let me know if it's not clear what I mean.